test/integration/spec/util/insightspublisher.js

```diff
@@ -51,7 +51,11 @@ describe('InsightsPublisher (@unstable: JSDK-2761)', function() {
             publisher.once('connected', () => reject(new Error('Unexpected connect')));
             publisher.once('disconnected', resolve);
           });
-          assert(error instanceof Error);
+          assert(error instanceof Error, `unexpected error ${error}`);
+          if (!(error instanceof Error)) {
```

Comment:
@makarandp0 Don't you mean:
```suggestion
          if (error instanceof Error) {
```

Reply:
that is intentional. 
We do expect `error` to be `instanceof Error`. we need this additional log line only in case its not to see what's going on.